eng/tools/versioning/VersionUtils.js

```diff
@@ -57,8 +58,13 @@ function updateChangelog(
     "--ReplaceLatestEntryTitle:$" + replaceLatestVersionTitle
   ];
   if (releaseDate != null) {
-    args.push(releaseDate);
+    args.push("--ReleaseDate:" + releaseDate);
   }
+
+  if (changelogPath != null) {
+    args.push("--ChangelogPath:" + changelogPath);
```

Comment:
I don't see this option on the script https://github.com/Azure/azure-sdk-tools/blob/master/eng/common/scripts/Update-ChangeLog.ps1. Is that part of another PR?

Reply:
Yes am going to add that. This is just preparing for that change. Basically am testing how it will work in each repo before updating the common script